lib/theme-and-appearance.zsh

```diff
@@ -25,7 +25,7 @@ if [[ "$DISABLE_LS_COLORS" != "true" ]]; then
       (( $+commands[dircolors] )) && eval "$(dircolors -b)"
     fi
 
-    ls --color -d . &>/dev/null && alias ls='ls --color=tty' || alias ls='ls -G'
+    ls --color -d . &>/dev/null && alias ls='ls --color=tty' || ls -G . &>/dev/null && alias ls='ls -G'
```

Comment:
This should be:

`ls --color -d . &>/dev/null && alias ls='ls --color=tty' || (ls -G . &>/dev/null && alias ls='ls -G')`

Note the parenthesis around the second test. This should fix the various ls colors issue that people are seeing.